src/modules/Profile/components/FavoriteCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { Divider, Modal, Typography } from "antd";
+import { ExclamationCircleOutlined } from "@ant-design/icons";
+
+import { TrashIcon } from "icons";
+import { useAppDispatch } from "hooks/useRedux";
+import { removeFromFavoriteList } from "redux/features/favorite.slice";
+import { routerPaths } from "routers/router.paths";
+import {
+  FavoriteCardContainer,
+  FavoriteCardDetail,
+  FavoriteCardStatus
+} from "../Profile.styled";
+
+const { Title, Text } = Typography;
+
+interface FavoriteCardProps {
```

Comment:
Should Pass down ID only ---> The rest (name, img) can be find in global store

Reply:
Handle as you mention is ok. but I will write 1 function to filter get object include (name, img). I think 2 ways have the same complexity

Reply:
As code blocks go bigger, lesser the Props will need to pass